oak-segment-azure/src/test/java/org/apache/jackrabbit/oak/segment/azure/v8/AzureSegmentStoreV8Test.java

```diff
@@ -70,7 +72,7 @@ public class AzureSegmentStoreV8Test {
 
     private static final EnumSet<SharedAccessBlobPermissions> READ_ONLY = EnumSet.of(READ, LIST);
     private static final EnumSet<SharedAccessBlobPermissions> READ_WRITE = EnumSet.of(READ, LIST, CREATE, WRITE, ADD);
-    private static final ImmutableSet<String> BLOBS = ImmutableSet.of("blob1", "blob2");
+    private static final Set<String> BLOBS = ImmutableSet.of("blob1", "blob2");
```

Comment:
What about the right side of the assignment?

Reply:
fixed in https://github.com/apache/jackrabbit-oak/pull/2212/commits/89ac490e9d6ce011ce9ccecfaf7b8adb3b3f24b1

Reply:
I still see ImmutableSet.of(...) there.

Reply:
Sorry, I misread you comment in the first place. I have reverted the `left side` to previous value, cause I would be handling the `ImmutableSet.of` changes in next jira: OAK-11646